_posts/2022-07-28-Siti-servizi-comunali-semplici-grazie-al-modello.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+lang: it
+excerpt_separator: <!-- MORE -->
+title: "Siti e servizi comunali a misura di cittadino grazie al modello aggiornato di Designers Italia"
+subtitle: "Pubblicata la prima versione delle risorse operative per realizzare esperienze digitali semplici e accessibili"
+date: 2022-07-28 09:00:00 +0200
+pinned: false
+nodate: false
+homeBottomLink: false
+homeHideCategories: true
+date_prepend: "Articolo"
+categories:
+  - cittadini
+  - pubblica amministrazione
+  - comuni
+---
+
+<!-- MORE -->
+Siti e servizi pubblici digitali sempre più a misura di cittadino. E’ questo lo spirito con il quale, negli scorso mesi. il team di Designers Italia ha affrontato l’aggiornamento delle risorse operative messe a disposizione dei Comuni italiani per realizzare o aggiornare il proprio sito web e i servizi digitali. 
```

Comment:
```suggestion
Siti e servizi pubblici digitali sempre più a misura di cittadino. È questo lo spirito con il quale, negli scorsi mesi, il team di Designers Italia ha affrontato l’aggiornamento delle risorse operative messe a disposizione dei Comuni italiani per realizzare o aggiornare il proprio sito web e i servizi digitali. 
```

Reply:
fixed in https://github.com/italia/designers.italia.it/pull/367/commits/5b6c97284382cbfa21072b5cd61c057d50bb5075